protostar/cairo/contract_class.py

```diff
@@ -0,0 +1,43 @@
+import json
+from pathlib import Path
+
+from starkware.starknet.core.os.contract_class.class_hash import compute_class_hash
+from starkware.starknet.core.os.contract_class.compiled_class_hash import (
+    compute_compiled_class_hash,
+)
+from starkware.starknet.services.api.contract_class.contract_class import (
+    ContractClass,
+    CompiledClass,
+)
+
+
+def make_contract_class(sierra_compiled: str) -> ContractClass:
+    sierra_compiled_dict = json.loads(sierra_compiled)
+    sierra_compiled_dict.pop("sierra_program_debug_info", None)
+    sierra_compiled_dict["abi"] = json.dumps(sierra_compiled_dict["abi"])
+
+    return ContractClass.load(sierra_compiled_dict)
+
+
+def make_compiled_class(casm_compiled: str) -> CompiledClass:
+    return CompiledClass.loads(casm_compiled)
+
+
+def compute_class_hash_from_sierra_code(sierra_compiled: str, output_path: Path):
+    contract_class = make_contract_class(sierra_compiled)
+    class_hash = compute_class_hash(contract_class)
+
+    with open(output_path, mode="w", encoding="utf-8") as output_file:
+        output_file.write(f"{hex(class_hash)}")
```

Comment:
I don't really like this approach of always writing to file when computing these. Write should be done separately in the methods that require it.

Reply:
I think we can leave writing to `output_path` here, but make it optional. This way, it will behave similarly to functions from `cairo_bindings`.

Reply:
That's better but I still would prefer if writing to file was moved entirely to the module needing it. 

I'm assuming writing output of this command to file is needed only in the single place of the library.